.vscode/extensions.json

```diff
@@ -6,4 +6,4 @@
 		"ban.spellright",
 		"eamodio.tsl-problem-matcher"
 	]
-}
+}
```

Comment:
Is there a rule for enforcing new lines at the end of files? Is [that](https://palantir.github.io/tslint/rules/eofline/) it? I don't feel comfortable without them.